marshal/rationalizer.go

```diff
@@ -182,11 +182,11 @@ func (c *KafkaCluster) releaseClaim(msg *msgReleasingPartition) {
 	topic.lock.Lock()
 	defer topic.lock.Unlock()
 
-	log.Infof("[%s:%d] Client %s releasing", msg.Topic, msg.PartID, msg.ClientID)
```

Comment:
Yeah, sorry. I think this was debugging logging left in from development of checkpointing. RIP it.